addons/addon-base-workflow/packages/base-workflow-core/lib/schema/create-wf-assignment.js

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { idRegex } = require('@amzn/base-services/lib/helpers/constants');
+
+const schema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    id: {
```

Comment:
Test id validation.

There are no tests for any of these services that use id validation: 
- workflow-assignment-registry-service.js
- workflow-assignment-service.js

Reply:
created Ticket for missing unit testing in services CHAM-212